add Content Panels for Musicvideos in SearchEngine

marduklev opened the pull request:
- find titles by searchstring
- find artist by search string

marduklev committed:
add Content Panels for Musicvideos in SearchEngine

- find titles by searchstring
- find artist by search string

jurialmunkey commented on 1080i/Custom_1138_Search.xml:
```diff
@@ -240,7 +240,50 @@
+                <include content="View_Cast_Toplabel">
+                    <param name="groupid" value="7516" />
+                    <param name="id" value="5016" />
+                    <param name="label" value="$INFO[Container(5016).NumItems,$LOCALIZE[20389] by (,)]" />
```
Can you localize the "by" on this line?

jurialmunkey reviewed (changes requested):
Looks good! Can you just localize the "by" on line 269 and then I will merge. Thanks 😃

jurialmunkey commented:
Hi. Just wondering if you were going to make the above requested change?

marduklev commented:
Sorry for late response, will check next monday.
msgctxt "#1443"
msgid "of"
msgstr "von"


I need also to check the artwork image

jurialmunkey commented:
Not a problem. No need to rush - was just making sure you hadn't missed my comments.

marduklev commented:
ok, unsure if it'll cause wrong it'll cause "grammar issues" if it'll use that 

 - $INFO[Skin.String(SearchTerm),$LOCALIZE[20391] - $LOCALIZE[133] $LOCALIZE[21400]  &quot; , &quot; ] $INFO[Container(5001).NumItems, (,)]

music videos - artist contains "searchstring" (numitems)

or if i should do pr , with a predefined label in language strings(.po)
 - $INFO[Skin.String(SearchTerm),$LOCALIZE[20389] $LOCALIZE[31432] &quot;[CAPITALIZE],[/CAPITALIZE]&quot;]$INFO[Container(5013).NumItems, (,)]
 
 = Music viedeos - "by Artist named" 'Searchstring' 

 - #: /1080i/Custom_1138_Search.xml
 - msgctxt "#31432"
 - msgid "By Artist Named"
 - msgstr ""

jurialmunkey commented:
Grammatical issues might be a problem, so I've been trying to keep it as simple as possible. 
e.g. For Albums by Artist I used the format: "Albums by 'Artist'"
The "Albums by" is localized as one string.

Maybe this would be best:
Music Videos by 'SearchString'
"Music Videos by" as one localized string.

That way, if it is a complete string, then there is at least some control in the translation for phrasing.

marduklev committed:
Update Custom_1138_Search.xml

- switch to list item icon as Fanart , unsure if you want use variable for that (i didnt look deep in your code)
- use localiced string for music videos by artist named 'searchstring'

marduklev commented:
- i think its done, at least , you should just check if you really want ListItem.Icon, instead of an variable for fanart

# e.g.
```
<value condition="!String.IsEmpty(ListItem.Art(artist.landscape))">$INFO[ListItem.Art(artist.landscape)]</value>
	<value condition="!String.IsEmpty(ListItem.Art(albumartist.landscape))">$INFO[ListItem.Art(albumartist.landscape)]</value>
	<value condition="!String.IsEmpty(ListItem.Art(landscape))">$INFO[ListItem.Art(landscape)]</value>
	<value condition="!String.IsEmpty(ListItem.Art(artist.fanart))">$INFO[ListItem.Art(artist.fanart)]</value>
	<value condition="!String.IsEmpty(ListItem.Art(albumartist.landscape))">$INFO[ListItem.Art(albumartist.fanart)]</value>
	<value condition="!String.IsEmpty(ListItem.Art(fanart))">$INFO[ListItem.Art(fanart)]</value>
	<value>$INFO[ListItem.Icon]</value>
```

jurialmunkey commented:
I made a few small changes, but everything has been merged.
Changed artwork to square as musicvideos default scraper scrapes album covers as ListItem.Icon and there doesn't seem to be anyway to get the actual video thumbnail (which is what I would actually prefer).
Changed the "by artist named" string to be one string "Music Videos by Artist Named" rather than two combined.
